src/util/webhooks.ts

```diff
@@ -87,3 +87,15 @@ export const emitWebhookFailedMetrics = (webhookName: string) => {
 	statsd.increment(metricWebhooks.webhookFailure, tags);
 
 }
+
+export const emitWebhookPayloadMetrics = (webhookName: string, size: number) => {
+	const tags = {
+		name: webhookName,
+		payloadSize: size.toString()
+	};
+	statsd.histogram(
```

Comment:
So, hence the purpose is to identify how many webhooks are there with a pyload > 128KB 

we should use a bucketed histogram metric here (See how it is done for the latencies metrics, we hse gsd_histogram parameter where we specify buckets sizes)

Reply:
Added bucket sizes.